### PR TITLE
bumped k8s to v0.19.3 and controller-runtime to v0.6.2

### PR DESCRIPTION
**Pull request by dannyzaken**
updated dependencies as an initial step before migrating to operator-sdk v1

Signed-off-by: Danny Zaken <dannyzaken@gmail.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -24,49 +24,53 @@ require (
...
+	k8s.io/gengo v0.0.0-20200428234225-8167cfdcfc14
+	k8s.io/kube-openapi v0.0.0-20200805222855-6aeccd4b50c6
+	k8s.io/kubectl v0.18.2
 	nhooyr.io/websocket v1.7.4
-	sigs.k8s.io/controller-runtime v0.5.2
+	sigs.k8s.io/controller-runtime v0.6.3
 	sigs.k8s.io/yaml v1.2.0
 )
 
 // Pinned to kubernetes-1.17.4
```
Comment (liranmauda) [inline go.mod:48]:
```suggestion
// Pinned to kubernetes-1.19.3
```

File: go.mod [hunk 2]
```diff
@@ -24,49 +24,53 @@ require (
...
 replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.2+incompatible // Required by OLM
```
Comment (liranmauda) [inline go.mod:50]: Do we still need this?

Comment (dannyzaken) [reply go.mod:50]: seems like we do. didn't work on removing it now

**Review by guymguym — COMMENTED** [review]
Does this PR change our minimal kubernetes version supported?

https://github.com/noobaa/noobaa-operator/blob/6672a508a4237180d9b84ec7e1869c2d57dc8754/deploy/olm/noobaa-operator.clusterserviceversion.yaml#L19

**Comment by nimrod-becker** [comment]
@dannyzaken have you performed the testing you wanted to ?

**Review by nimrod-becker — APPROVED** [review]

**Pull request merged**